lab_3_keywords_textrank/main.py

```diff
@@ -136,7 +149,12 @@ def encode(self, tokens: tuple[str, ...]) -> Optional[tuple[int, ...]]:
                 sequence of integer tokens
         In case of empty tokens input data, None is returned
         """
-        pass
+        if not tokens:
+            return None
+
+        self._learn_indices(tokens)
+        encoded_tokens = tuple(self._word2id[token] for token in tokens)
```

Comment:
again - you do not need create the `encoded_tokens` variable

Reply:
done